### ISSUE TITLE
radionuclide JSON format

### ISSUE THREAD

**Issue by KrisThielemans** [UCL/STIR#1027]
At present we have a `properties` field like this
```
    "^90^Yttrium":

    {  "modality": 
                
      {   "nucmed":
        {  "properties": [
                 {
                  "kev": 150,
                  "BRatio": 0.99983,
                  "half_life": 230549.76
                 }       ]
        }, 
         "PET":
        {  "properties": [
                 {
                  "kev": 511,
                  "BRatio": 0.0000319,
                  "half_life": 230549.76
                  }       ]
        }
      }
    }
```
@danieldeidda I'm not sure why we're having this level. It makes more sense to me to have a list of records with all properties, like
```
    "^90^Yttrium": [

    {  "modality":   "nucmed",
        "kev": 150,
        "BRatio": 0.99983,
         "half_life": 230549.76
     } ,
    {  "modality":   "PET",
        "kev": 511,
        "BRatio": 0.0000319,
        "half_life": 230549.76
     }
   ]
```
This makes it just a list of all decays.

This would mean some change in the logic of the reading code, but that seems feasible. Of course, I haven't looked into JSON at all, so I'm not sure if this works.

In addition, I'm a bit hesitant on using `kev` as opposed to `keV`, as well `BRatio` vs `branching_ratio`.

To get some guidance, I had a look at the BIDS standard. This doesn't seem to have related fields, [https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#radiochemistry](except for `TracerRadionuclide`) which gives an example `C11`, but I think relying on DICOM is better.

**Comment by danieldeidda** [comment]
I don't remember the details but I think  the structure you propose is the one we had originally thought, however it was  challenging to implement it and we were having issues to discriminate between modalities.  Modality and radionuclide name are the two keywords that need to be read from the interfile to select the radionuclide used for PET or SPECT. The difficulty was particularly for Y90 

for the naming kev and BRatio that should be easy to change

**Comment by KrisThielemans** [comment]
fixed in #1026
